libs/dh/imbalance-prices/shell/src/lib/drawer/dh-drawer.component.ts

```diff
@@ -162,9 +177,12 @@ export class DhImbalancePricesDrawerComponent {
       message: translate('shared.downloadStart'),
     });
 
+    const formatter = new Intl.DateTimeFormat(this.locale, { month: 'long', year: 'numeric' });
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const formattedDate = formatter.format(this.imbalancePrice()!.name);
+
     const fileOptions = {
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      name: 'imbalance-prices-' + format(this.imbalancePrice()!.name, 'MMMM-yyyy'),
+      name: 'imbalance-prices-' + formattedDate,
```

Comment:
suggestion: let's reuse `dayjs` to format the date. It's been used everywhere else anyway.